xrootd/xrootd_config.go

```diff
@@ -334,6 +344,10 @@ func CheckCacheXrootdEnv(exportPath string, server server_utils.XRootDServer, ui
 		}
 	}
 
+	if viper.GetString("Cache.PSSOrigin") == "" {
+		return "", errors.New("Cache pss.origin url empty")
```

Comment:
It's unlikely the user knows what "Cache pss.origin url empty" is or how to solve it.

How about this:
```suggestion
		return "", errors.New("One of Federation.DiscoveryUrl or Federation.DirectorUrl must be set to configure a cache")
```